.github/workflows/_reusable_build_package.yml

```diff
@@ -66,6 +68,8 @@ jobs:
     outputs:
       package_path: ${{ steps.package.outputs.path }}
     steps:
+      - name: Debug GitHub Token
+        run: echo "Token is ${{ secrets.GH_CI_TOKEN }}"
```

Comment:
This should be removed, and we should change the token afterwards

Reply:
Addressed